en/components/grid-column-hiding.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Grid Column Hiding - Grid Feature
+_description: The Grid Column Hiding is a mechanism to hide and show columns of the Grid.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Grid Column Hiding
+
+Use the Grid Column Hiding to hide certain columns from the Grid. The easiest way to provide an interface for making columns visible again is by adding a Toolbar, which by default contains a button (combined with combo) that lets you configure the visibility of columns for the whole grid. The Grid Column Hiding is identical to the [Ignite UI for Angular Grid Column Hiding Feature](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/column_hiding.html)
+
+### Grid Column Hiding Demo
+
+<img class="responsive-img" src="../images/grid_column_hiding_demo.png" srcset="../images/grid_column_hiding_demo@2x.png 2x" />
+
+### Toolbar and Header Cell
+
+The Grid Column Hiding is achieved by adding a Toolbar with a Column Hiding action that enables the feature for the whole grid but does not tell it which columns to show or hide. An alternative approach is to configure the `Column Hiding` override on the Grid Header Cell and set it to Hidden to make a certain column invisible, the override's default value is **Visible**.
```

Comment:
I think the it and to are redundant: The Grid Column Hiding is achieved by adding a Toolbar with a Column Hiding action that enables the feature for the whole grid but does not tell **it** which columns **to** show or hide. 

Feedback: **does not tell** sounds a little bit strange when reading.

Reply:
I agree so I rephrased the whole paragraph